src/main/resources/changelog.xml

```diff
@@ -32,8 +32,6 @@
             <column name="animal_id" type="long">
             </column>
         </createTable>
-    </changeSet>
```

Comment:
you cannot delete a change set once it is there

Reply:
Yeah, I think I need a quick tutorial about the use of changeset.

Reply:
Yeah, we can talk about that later